app/isomorphic/components/layouts/footer/styles.m.css

```diff
@@ -1,13 +1,47 @@
 .footer {
   clear: both;
-  text-align: center;
-  font-size: 20px;
-  background-color: black;
-  color: white;
-  padding: 20px 0;
+  border-top: 1px solid black;
+  padding: 15px 0;
+  display: flex;
+  justify-content: space-around;
+  position: absolute;
+  bottom: 0;
+  width: 968px;
```

Comment:
why is this hardcoded ?

Reply:
Just to make it same size of the container. 
However this will have the same size as header so these styles gets changed after merging